src/vivarium_data_requests/jun_17_laura_lamberti/requirements.txt

```diff
@@ -0,0 +1,6 @@
+drmaa
```

Comment:
Did you generate this file by hand?

I would suggest, once you can build the artifact, running `pip freeze > requirements.txt` to make sure you capture the full environment.

Reply:
Also didn't you mention you ran into issues where some of the data needed tables 3.4.0 and some needed a newer version?

Reply:
the tables thing is actually only relevant for the old request, so that will go into that section. I'll make this using freeze that'll be better